### PR TITLE
chore: switch from travis to gh actions

### PR DESCRIPTION
**Pull request by open-dynaMIX**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+          echo "UID=$(id --user)" > .env
+          docker-compose up -d --build
+      - name: Lint the code
+        run: |
+          docker-compose exec -T caluma black --check .
+          docker-compose exec -T caluma flake8
+          docker-compose exec -T caluma ./manage.py makemigrations --check --dry-run --no-input
+          docker-compose exec -T caluma reuse lint
+      - name: Run pytest
+        run: |
```
Comment (anehx) [inline .github/workflows/tests.yml:25]: Not necessary, inline the following line

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+          docker-compose exec -T caluma pytest --no-cov-on-fail --cov --create-db -vv
+  pipenv-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
```
Comment (anehx) [inline .github/workflows/tests.yml:30]:
```suggestion
      - uses: actions/checkout@v2
```

File: .github/workflows/tests.yml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+          PIPENV_IGNORE_VIRTUALENVS: 1
+        run: |
+          pip install pipenv
+          python3 setup.py pipenv
+      - name: Run pytest
+        run: pipenv run pytest
+  package-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
```
Comment (anehx) [inline .github/workflows/tests.yml:46]:
```suggestion
      - uses: actions/checkout@v2
```

File: .github/workflows/tests.yml [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+          pip install setuptools wheel
+          python setup.py bdist_wheel
+          pip install dist/caluma-*.whl
+          pip install -r requirements-dev.txt
+      - name: Run pytest
+        run: pytest --no-cov-on-fail --cov --create-db -vv
+  gitlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
```
Comment (anehx) [inline .github/workflows/tests.yml:67]:
```suggestion
      - uses: actions/checkout@v2
```

**Review by anehx — CHANGES_REQUESTED** [review]

File: .github/workflows/tests.yml (hunk 1 above)
Comment (open-dynaMIX) [reply .github/workflows/tests.yml]: Done

File: .github/workflows/tests.yml (hunk 2 above)
Comment (open-dynaMIX) [reply .github/workflows/tests.yml]: Done

File: .github/workflows/tests.yml (hunk 3 above)
Comment (open-dynaMIX) [reply .github/workflows/tests.yml]: Done

File: .github/workflows/tests.yml (hunk 4 above)
Comment (open-dynaMIX) [reply .github/workflows/tests.yml]: Done

**Review by anehx — APPROVED** [review]

**Comment by anehx** [comment]
Good work! :100: :fire:

**Pull request merged**